outlines/models/exllamav2.py

```diff
@@ -162,71 +256,66 @@ def exl2(
         from exllamav2 import (
             ExLlamaV2,
             ExLlamaV2Cache,
-            ExLlamaV2Cache_8bit,
             ExLlamaV2Cache_Q4,
             ExLlamaV2Config,
+            ExLlamaV2Tokenizer,
         )
+        from exllamav2.generator import ExLlamaV2DynamicGenerator
         from transformers import AutoTokenizer
+
     except ImportError:
         raise ImportError(
-            "The `exllamav2`, `transformers` and `torch` libraries needs to be installed in order to use `exllamav2` models."
+            "The `exllamav2` and `torch` libraries needs to be installed in order to use `exllamav2` models."
```

Comment:
Transformers needs to be installed as well.

Reply:
Good point fixing now.